src/ripple/app/ledger/tests/common_ledger.cpp

```diff
@@ -435,28 +435,61 @@ trust(TestAccount& from, TestAccount const& issuer,
 void
 close_and_advance(Ledger::pointer& ledger, std::shared_ptr<Ledger const>& LCL)
```

Comment:
`jtx::advance` only calls into _common_ledger_ for simplicity and to avoid another copy/paste. If we want to make the push to deprecate _common_ledger_, we should *move* that code into `jtx::advance`. Additionally, the intention of `jtx::advance` was for specialized usages to create local overrides instead of proliferating a bunch of global overrides.